hybris.c

```diff
@@ -30,26 +30,48 @@
 #define LIB_DROID_MEDIA_PATH "libdroidmedia.so"
 #endif
 
-void *android_dlopen(const char *name, int flags);
-void *android_dlsym(void *handle, const char *name);
+#ifndef LIBHYBRIS_PATH
+#define LIBHYBRIS_PATH "libhybris-common.so.1"
+#endif
+
+static void *(*_android_dlopen)(const char *name, int flags);
+static void *(*_android_dlsym)(void *handle, const char *name);
```

Comment:
Should these be initialized to NULL pointers to make sure they are always NULL before the init checks in the code?

Reply:
They are static variables, they must be initialized with zeros by the compiler. Do you think I need to do this implicitly?

Did you update the hardware dependent part of droidmedia? I guess it cannot resolve `_droid_media_init` since I changed its signature.

Reply:
Perhaps this was the wrong decision. Perhaps it would be better to introduce something like `droid_media_is_supported ()`?

Reply:
I think I forgot to build the other part, need to try with that, missed that it had a change also.

Reply:
I will report back after the build is finished I can test again. I think we don't need any additional function. We might consider checking the newly added return value of droid_media_init in gst/plugin.c of gst-droid?

Reply:
After some checking there is no need to initialize the static variable in the code since compiler will do that automatically.

Reply:
> I will report back after the build is finished I can test again. I think we don't need any additional function. We might consider checking the newly added return value of droid_media_init in gst/plugin.c of gst-droid?

Thank you. I will consider adding the check to gst-droid and update the PR.

Reply:
I tested the changes on a device and now those worked.